docker/docker-compose.yml

```diff
@@ -6,6 +6,8 @@ services:
       - dbdata:/var/lib/mongodb
     networks:
       - webnet
+    ports:
+      - 27017:27017
```

Comment:
It exposes it from the Docker container to the Proxmox VM ; not to the outside world (we would need to open port 27017 on the machine for this). The reason we did this is because the Robotoff container (same machine, separate Proxmox container) needs to access this db as well.

Reply:
+ I agree that eventually we'll still have to put some credentials through GitHub secrets on the MongoDB deployment

Reply:
Would this work with Robotoff if we used 
```suggestion
      - "127.0.0.1:27017:27017"
```
instead? Or possibly even with an environment variable for the interface? The default binding `27017:27017` publishes the port on all interfaces, which makes it somewhat easy to accidentally open the port.